apps/src/code-studio/districtDropdown/SchoolInfoBox.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+// import i18n from '@cdo/locale'; // TODO
+
+function SchoolCountry(props) {
+  return (
+    <div className="itemblock">
+      <div className="labelblock">School Country</div>
+      <select className="form-control fieldblock" id="school-country" name="user[school_info_attributes][country]" type="select" value={props.country} onChange={props.onChange}>
+        <option value="US">United States</option>
+        <option value="AD">Andorra</option>
+        <option value="AE">United Arab Emirates</option>
+      </select>
+    </div>
+  );
+}
+
+SchoolCountry.propTypes = {
+  country: React.PropTypes.string,
+  onChange: React.PropTypes.func
+};
+
+function SchoolType(props) {
+  return (
+    <div className="itemblock">
+      <div className="labelblock">School Type</div>
+      <select className="form-control fieldblock" id="school-type" name="user[school_info_attributes][school_type]" type="select" value={props.schoolType} onChange={props.onChange}>
+        <option disabled="true" value=""/>
```

Comment:
Make a const array with all of these values, then do arrays.map() to generate these options